### DISCUSSION TITLE
Tryinattempt to disable registration of a new user - but registration continuesg to disable user registration

### DISCUSSION THREAD

**Discussion by Interfud** [dani-garcia/vaultwarden#4111] [Q&A]
Release 1.30.1 (Ubuntu 22.04 server)
docker run -d --name vaultwarden -e ADMIN_TOKEN=******* -v /vw-data/:/data/  -e WEBSOCKET_ENABLED=true -e SIGNUPS_ALLOWED=false -e INVITATIONS_ALLOWED=false --restart unless-stopped -p 80:80 vaultwarden/server:latest
.
![Снимок2](https://github.com/dani-garcia/vaultwarden/assets/94401324/d68d739c-04f9-4d61-b415-9606b08a10e3)

The deadline for registering a new user continues to appear
Registration continues
A new user appears(.
It's not very dangerous for me.

**Comment by RealOrangeOne** [comment]
It looks like you have `SIGNUPS_ALLOWED=false` in your environment variables. I believe environment variables take precedence over what you set in the admin, so you'll probably need to remove the environment variable if you want to control it through the admin, or stop using the admin and define everything through environment variables.

**Reply by BlackDex** [reply]
It is actually the other way around.
Everything done in the admin interface overrules the ENV variables.
So, with that checkbox enabled, it allows new signups.

So, there probably is a `config.json` which has settings, and that overrules everything.

**Comment by Interfud** [comment]
file config.v2.json
![Снимок3](https://github.com/dani-garcia/vaultwarden/assets/94401324/94e9eccf-da4c-4a73-a07d-5b993d860b75)

**Reply by BlackDex** [reply]
That is the wrong config. There probably is a `config.json` in your `/vw-data/` directory.

**Comment by Interfud** [comment]
Thank you. It was fast(
